src/openms/source/ANALYSIS/ID/SiriusAdapterAlgorithm.cpp

```diff
@@ -44,99 +43,471 @@
+                              "To use this parameter, setting the feature_only flag is necessary")
+                ).withMinInt(1);
+
+      parameter(
+                  OpenMSName("precursor_mz_tolerance"),
+                  DefaultValue(10),
+                  Description("Tolerance window for precursor selection (Feature selection in regard to the precursor)")
+                );
+
+      parameter(
+                  OpenMSName("precursor_mz_tolerance_unit"),
+                  DefaultValue("ppm"),
+                  Description("Unit of the precursor_mz_tolerance")
+               ).withValidStrings({"Da", "ppm"});
+
+      parameter(
+                  OpenMSName("precursor_rt_tolerance"),
+                  DefaultValue(5),
+                  Description("Tolerance window (left and right) for precursor selection [seconds]")
+               );
+
+      parameter(
+                  OpenMSName("isotope_pattern_iterations"),
+                  DefaultValue(3),
+                  Description("Number of iterations that should be performed to extract the C13 isotope pattern. "
+                              "If no peak is found (C13 distance) the function will abort. "
+                              "Be careful with noisy data - since this can lead to wrong isotope patterns")
+                );
+
+      flag(
+            OpenMSName("feature_only"),
+            Description("Uses the feature information from in_featureinfo to reduce the search space to MS2 "
+                        "associated with a feature")
+          );
+
+      flag(
+            OpenMSName("no_masstrace_info_isotope_pattern"),
+            Description("Use this flag if the masstrace information from a feature should be discarded "
+                       "and the isotope_pattern_iterations should be used instead")
+          );
+    }
+
+    void SiriusAdapterAlgorithm::Project::parameters()
+    {
+      // This will be called internal using the "out_project_space" parameter (SiriusAdapter)
+      // -o, -p, --output, --project-space=<projectSpaceLocation>
+      // Specify project-space to read from and also write to if
+      // nothing else is specified. For compression use the File
+      // ending .zip or .sirius
+
+      // This should not be changes since the mztab writers/readers are depend on the default naming schema on it.
+      // --naming-convention=<projectSpaceFilenameFormatter>
+      //    Specify a naming scheme for the  compound
+      // directories ins the project-space. Default %index_%filename_%compoundname
+
+      // This will be called internal using the preprocessed .ms file
+      // -i, --input=<input>
+      // Input for the analysis. Ths can be either preprocessed mass
+      // spectra in .ms or .mgf file format, LC/MS runs in .mzML/.
+      // mzXml format or already existing SIRIUS project-space(s)
+      // (uncompressed/compressed).
+
+      // --maxmz=<maxMz>
+      // Just consider compounds with a precursor mz lower or equal
```

Comment:
Remove the comments where they are the same in the description.

Reply:
These are basically the descriptions from the SIRIUS tool, should I still remove them?

Reply:
Yeah I know. I just thought, since you copy them word for word into the parameter description, I don't see a need.
The parameter name is also always just the OpenMS name with -- right?

Reply:
Yes, you are right. I guess then they are not needed there, or at least the description.